wordpress-mobile/WordPress-Android
# [Reader] Implement tags feed card actions (part 1)

Fixes #20670 

Part 1 of the "Tags" feed card actions implementation. Actions implemented in this PR:
- Open site details.
- Open post details.

Next PR (part 2):
- Like.
- Overflow menu.

-----

## To Test:

1. Instal JP and log in.
2. Enable `reader_tags_feed` feature flag and restart the app.
3. Open Reader and select "Tags" feed with at least one tag subscribed.
4. 🔍 Tap the site name in any post: the site details should be opened.
5. 🔍 Tap either the post title, excerpt or image: the post details should be opened.
6. 🔍 When navigating back from the screens opened in steps 4 and 5 you should be back to the "Tags" feed.

-----

## Regression Notes

1. Potential unintended areas of impact

    - None

2. What I did to test those areas of impact (or what existing automated tests I relied on)

    - Manual testing and updated unit tests

3. What automated tests I added (or what prevented me from doing so)

    - Updated `ReaderTagsFeedViewModelTest.kt` and `ReaderTagsFeedUiStateMapperTest.kt`

-----

## PR Submission Checklist:


-----

## Testing Checklist (strike-out the not-applying and unnecessary ones):
